### DISCUSSION TITLE
How can I change where my music is downloaded?

### DISCUSSION THREAD

**Discussion by Skaptrus** [zotify-dev/zotify#73] [Q&A]
When I download my music, by default, it is downloaded into the music folder. However, I want it to be downloaded into another hard drive. How can I change the download location?

**Comment by CipherFrame** [comment]
Go to C:\Users\<USERNAME>\AppData\Roaming\Zotify and open config.JSON in a text editor. Change the following parameters with your required download location:

  "ROOT_PATH": "\\\\your\\download\\location\\here",
   "ROOT_PODCAST_PATH": "\\\\your\\download\\location\\here",

**Reply by saltywhewnet** [reply]
it gives me error:   C:\Users\salty>: The term 'C:\Users\salty>' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

This is what i entered :  "ROOT_PATH": "C:\Users\salty\salty Dropbox\salty\PC",

**Comment by CipherFrame** [comment]
When defining paths in JSON files like config.JSON, it's essential to use double backslashes (\\) to separate directories, and you should enclose the entire path in double quotes if it includes spaces. Here's how you can correctly format the path in the JSON file:

"ROOT_PATH": "C:\\Users\\salty\\salty Dropbox\\salty\\PC".

Make sure to check the path carefully to ensure it points to a valid directory on the system. If the directory doesn’t exist, you’ll need to create it or specify a different path that already exists.

**Comment by PielieChicken** [comment]
So its like this: 

"ROOT_PATH": "D:\\downloads\\zotify",
"ROOT_PODCAST_PATH": "D:\\downloads\\podcasts"

**Comment by habuham** [comment]
Bumping this to leave my solution for anyone with the same problem.

I had this same issue, and I eventually figured it out. It worked when I formatted the paths as such:

    "ROOT_PATH": "D:/Zotify/Music",
    "ROOT_PODCAST_PATH": "D:/Zotify/Podcasts",

So use forward slashes and specify the drive.

Hope this helps someone.
